diff -ur oath-toolkit-2.6.2.orig/liboath/gl/tests/intprops.h oath-toolkit-2.6.2/liboath/gl/tests/intprops.h
--- oath-toolkit-2.6.2.orig/liboath/gl/tests/intprops.h	2016-08-27 13:15:06.000000000 +0200
+++ oath-toolkit-2.6.2/liboath/gl/tests/intprops.h	2019-03-14 13:02:37.921000000 +0100
@@ -230,11 +230,11 @@
    (e.g., A and B) have the same type as MIN and MAX.  Instead, they assume
    that the result (e.g., A + B) has that type.  */
 #if _GL_HAS_BUILTIN_OVERFLOW_WITH_NULL
-# define _GL_ADD_OVERFLOW(a, b, min, max)
+# define _GL_ADD_OVERFLOW(a, b, min, max)                                \
    __builtin_add_overflow (a, b, (__typeof__ ((a) + (b)) *) 0)
-# define _GL_SUBTRACT_OVERFLOW(a, b, min, max)
+# define _GL_SUBTRACT_OVERFLOW(a, b, min, max)                           \
    __builtin_sub_overflow (a, b, (__typeof__ ((a) - (b)) *) 0)
-# define _GL_MULTIPLY_OVERFLOW(a, b, min, max)
+# define _GL_MULTIPLY_OVERFLOW(a, b, min, max)                           \
    __builtin_mul_overflow (a, b, (__typeof__ ((a) * (b)) *) 0)
 #else
 # define _GL_ADD_OVERFLOW(a, b, min, max)                                \
diff -ur oath-toolkit-2.6.2.orig/libpskc/gl/intprops.h oath-toolkit-2.6.2/libpskc/gl/intprops.h
--- oath-toolkit-2.6.2.orig/libpskc/gl/intprops.h	2016-08-27 13:15:06.000000000 +0200
+++ oath-toolkit-2.6.2/libpskc/gl/intprops.h	2019-03-14 13:02:54.816000000 +0100
@@ -230,11 +230,11 @@
    (e.g., A and B) have the same type as MIN and MAX.  Instead, they assume
    that the result (e.g., A + B) has that type.  */
 #if _GL_HAS_BUILTIN_OVERFLOW_WITH_NULL
-# define _GL_ADD_OVERFLOW(a, b, min, max)
+# define _GL_ADD_OVERFLOW(a, b, min, max)                                \
    __builtin_add_overflow (a, b, (__typeof__ ((a) + (b)) *) 0)
-# define _GL_SUBTRACT_OVERFLOW(a, b, min, max)
+# define _GL_SUBTRACT_OVERFLOW(a, b, min, max)                           \
    __builtin_sub_overflow (a, b, (__typeof__ ((a) - (b)) *) 0)
-# define _GL_MULTIPLY_OVERFLOW(a, b, min, max)
+# define _GL_MULTIPLY_OVERFLOW(a, b, min, max)                           \
    __builtin_mul_overflow (a, b, (__typeof__ ((a) * (b)) *) 0)
 #else
 # define _GL_ADD_OVERFLOW(a, b, min, max)                                \
diff -ur oath-toolkit-2.6.2.orig/oathtool/gl/intprops.h oath-toolkit-2.6.2/oathtool/gl/intprops.h
--- oath-toolkit-2.6.2.orig/oathtool/gl/intprops.h	2016-08-27 13:15:06.000000000 +0200
+++ oath-toolkit-2.6.2/oathtool/gl/intprops.h	2019-03-14 12:59:53.022000000 +0100
@@ -230,11 +230,11 @@
    (e.g., A and B) have the same type as MIN and MAX.  Instead, they assume
    that the result (e.g., A + B) has that type.  */
 #if _GL_HAS_BUILTIN_OVERFLOW_WITH_NULL
-# define _GL_ADD_OVERFLOW(a, b, min, max)
+# define _GL_ADD_OVERFLOW(a, b, min, max)                                \
    __builtin_add_overflow (a, b, (__typeof__ ((a) + (b)) *) 0)
-# define _GL_SUBTRACT_OVERFLOW(a, b, min, max)
+# define _GL_SUBTRACT_OVERFLOW(a, b, min, max)                           \
    __builtin_sub_overflow (a, b, (__typeof__ ((a) - (b)) *) 0)
-# define _GL_MULTIPLY_OVERFLOW(a, b, min, max)
+# define _GL_MULTIPLY_OVERFLOW(a, b, min, max)                           \
    __builtin_mul_overflow (a, b, (__typeof__ ((a) * (b)) *) 0)
 #else
 # define _GL_ADD_OVERFLOW(a, b, min, max)                                \
diff -ur oath-toolkit-2.6.2.orig/pskctool/gl/intprops.h oath-toolkit-2.6.2/pskctool/gl/intprops.h
--- oath-toolkit-2.6.2.orig/pskctool/gl/intprops.h	2016-08-27 13:15:06.000000000 +0200
+++ oath-toolkit-2.6.2/pskctool/gl/intprops.h	2019-03-14 13:03:17.341000000 +0100
@@ -230,11 +230,11 @@
    (e.g., A and B) have the same type as MIN and MAX.  Instead, they assume
    that the result (e.g., A + B) has that type.  */
 #if _GL_HAS_BUILTIN_OVERFLOW_WITH_NULL
-# define _GL_ADD_OVERFLOW(a, b, min, max)
+# define _GL_ADD_OVERFLOW(a, b, min, max)                                \
    __builtin_add_overflow (a, b, (__typeof__ ((a) + (b)) *) 0)
-# define _GL_SUBTRACT_OVERFLOW(a, b, min, max)
+# define _GL_SUBTRACT_OVERFLOW(a, b, min, max)                           \
    __builtin_sub_overflow (a, b, (__typeof__ ((a) - (b)) *) 0)
-# define _GL_MULTIPLY_OVERFLOW(a, b, min, max)
+# define _GL_MULTIPLY_OVERFLOW(a, b, min, max)                           \
    __builtin_mul_overflow (a, b, (__typeof__ ((a) * (b)) *) 0)
 #else
 # define _GL_ADD_OVERFLOW(a, b, min, max)                                \
